cap_client/cli/repositories_cli.py

```diff
@@ -69,8 +69,8 @@ def upload(ctx, pid, url, webhook):
             event_type=webhook
         )
         click.echo('Repository {} saved in analysis {}.'.format(url, pid))
-        click.echo(json.dumps(response,
-                              indent=4))
+        if webhook:
+            click.echo(json.dumps(response, indent=4))
```

Comment:
let's not mix messages with json in responses, so users can fetch it programatically :)
so:
```suggestion
        if webhook:
            click.echo(json.dumps(response, indent=4))
        else:
            click.echo('Repository {} saved in analysis {}.'.format(url, pid))
```